src/board/board.ts

```diff
@@ -66,32 +73,7 @@ export type ActionLayout = {
    * the element as a percentage of the entire board's size.
    */
   gap?: number,
-} & (({
-  position?: 'inset'
-}) & (({
-  left?: number,
-  right?: never,
-  gap?: never,
-} | {
-  left?: never,
-  right?: number,
-  gap?: never,
-}) & ({
-  top?: number,
-  bottom?: never,
-  gap?: never
-} | {
-  top?: never,
-  bottom?: number,
-  gap?: never
-})) | {
-  position: 'beside' | 'stack'
-  left?: never,
-  right?: never,
-  top?: never,
-  bottom?: never,
-  gap?: number,
-});
```

Comment:
type was too complicated for jsdoc and i got frustrated and remove the add'l guards